### ISSUE TITLE
An XML documents issue (color)

### ISSUE THREAD

**Issue by delminskii** [chriskempson/base16-vim#149]
Hi everyone. I have an issue with XML tags - the opening and the closed ones have the different color (it's strange that I have normal colors in HTML). Please, take a look on the screenshot below:
![13](https://user-images.githubusercontent.com/6321554/27533438-45bca738-5a6c-11e7-87b8-76ca88b8da21.png)

Vim version:  `8.0.69`
OS: `ubuntu 14.04.1`
Terminal: `xterm(256)`

**Comment by mnarrell** [comment]
Is there any update here?  Is this working as designed?

**Comment by delminskii** [comment]
@mnarrell I've checked right now on my `nvim0.2.1-dev`  - the same.

**Comment by delminskii** [comment]
@mnarrell I think it was made intentionally as it IS in molokai (see [here](https://github.com/tomasr/molokai/issues/5), so this hack works here too
`hi def link xmlEndTag Function`

**Comment by mnarrell** [comment]
@delminskii Yes, this does work.  And I agree, this seems to be an intentional feature.  Thanks for your clarification.
